backend/api_gateway/carpool_request_endpoint/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from backend.api_gateway.carpool_request_endpoint.views \
+    import create_carpool_request
+from backend.api_gateway.carpool_request_endpoint.views \
+    import delete_carpool_request
+
+urlpatterns = [
+    path('create', create_carpool_request, name='create_carpool_request'),
+    path('cancel', delete_carpool_request, name='delete_carpool_request'),
```

Comment:
On REST API standard there should be only nonces on the path. So I think that 'create', 'cancel' should be integrated into one endpoint '' and whether the request is create or cancel is determined by method (POST, DELETE)